src/main/java/application/GetRelevantConversations.java

```diff
@@ -10,30 +10,23 @@
 import java.util.HashMap;
 
 public class GetRelevantConversations extends UserInteractor{
-    private User user;
     private ConversationQueue conversationQueue;
 
-    /**
-     * Initialize a new GetRelevantConversations.
-     */
-    GetRelevantConversations(User user){
-        this.user = user;
-
-    }
-
     /**
      * A request to be carried out by GetRelevantConversations.
      */
     public class GetRelevantConversationsRequest extends RequestModel{
         private OutputBoundary respondTo;
+        private User user;
         private int locationRadius;
 
         /**
          * Fills in this RequestModel's instance attributes.
          */
-        public void fillRequest(OutputBoundary respondTo, int locationRadius) {
+        public void fillRequest(OutputBoundary respondTo, User user, int locationRadius) {
```

Comment:
requests cannot require entities as attributes - dependencies cannot jump two layers in Clean Architecture.

Reply:
It can use a UserManager as an attribute though. I'd replace user with a UserManager I think.

Reply:
For somewhat reason I can't delete the binary file gradle wrapper.jar.